pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -176,7 +176,30 @@ func (r *ReconcileOperConfig) Reconcile(request reconcile.Request) (reconcile.Re
 	// is safe.
 	if prev != nil {
 		// Check if the operator is put in the 'Network Migration' mode.
-		if _, ok := operConfig.GetAnnotations()[names.NetworkMigrationAnnotation]; !ok {
+		if val, ok := operConfig.GetAnnotations()[names.NetworkMigrationAnnotation]; !ok || val != "start" {
+			// Prepare migration, set status but not apply objects during preparation phase.
+			if val == "prepare" {
```

Comment:
what if val is not prepare? should we log some error saying invalid value for migration?

Reply:
fixed.